### ISSUE TITLE
open browser window for more accessibility info doesn't work

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#213177]
In screen reader mode, focus a terminal. `Alt+f1`. See action at the bottom to open a link for more info. Follow instructions to try it, 🐛 doesn't work, view hides.

This action is also hard coded atm. it should be an `Action`.

![Screenshot 2024-05-21 at 12 29 11 PM](https://github.com/microsoft/vscode/assets/29464607/64afdd75-b6bb-44f9-8097-61c10a673427)

**Comment by meganrogge** [comment]
@andreamah could you toggle keyboard shortcuts troubleshooting or assign a new keybinding to see if that works? 

also see if changing `terminal.integrated.altClickMovesCursor` helps
